tool_quality_checks/quality_checks.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+""" TODO Docstring. """
+import os
+import argparse
+import logging
+
+from configparser import RawConfigParser
+from db import ThreediDatabase
+
+logger = logging.getLogger(__name__)
+
+
+def quality_checks(settings):
+    """Overall function for checking our model data"""
+
+    # get database connection
+    db = ThreediDatabase(settings)
+    # TODO: Make it only if install is on.
+    db.initialize_db()
+
+    # get v2_table_names
+    v2_table_names = db.select_table_names("v2%")
+    for table_name in v2_table_names:
+        if db.get_count(table_name) > 0:
+            print(table_name, db.get_count(table_name))
+            db.perform_checks_with_sql(settings, table_name, check_type="completeness")
+            db.perform_checks_with_sql(settings, table_name, check_type="quality")
+
+    db.populate_geometry_columns()
+
+
+def resolve_ini(custom_ini_file):
+    """
+    decide which ini file to use
+    """
+    # get default ini for testing purposes
+    default_ini_relpath = "test\\data\\instellingen_test.ini"
```

Comment:
Dit is met backslashes. Beter forward slashes gebruiken, python vertaalt dat wel. (Of os.path.join).